eddy/ui/ontology.py

```diff
@@ -1387,3 +1401,365 @@ def createTemplate(self):
                 <b><font color="#7E0B17">ERROR</font></b>:
                 Could not complete the export, see the System Log for details.
                 """)
+
```

Comment:
Questa logica sarebbe meglio fattorizzarla in uno (o più) OntologyLoaders, in modo da tenerla separata dal codice dell'interfaccia grafica